packages/twenty-server/src/modules/connected-account/auto-companies-and-contacts-creation/utils/get-unique-contacts-and-handles.util.ts

```diff
@@ -0,0 +1,18 @@
+import { uniq, uniqBy } from 'lodash';
+
+import { Contacts } from 'src/modules/connected-account/auto-companies-and-contacts-creation/types/contact.type';
+
+export function getUniqueContactsAndHandles(contacts: Contacts): {
```

Comment:
can you add a simple test in packages/twenty-server/src/modules/connected-account/auto-companies-and-contacts-creation/utils/__ tests__